cmd/amplifier/server/configuration.go

```diff
@@ -0,0 +1,46 @@
+package server
+
+import (
+	"fmt"
+
+	"github.com/spf13/viper"
+	"log"
+)
+
+// Config is used for amplifier configuration settings
+type Configuration struct {
+	Version          string
+	Build            string
+	Port             string
+	PublicAddress    string
+	EtcdEndpoints    []string
+	ElasticsearchURL string
+	NatsURL          string
+	DockerURL        string
+	DockerVersion    string
+	EmailKey         string
+	EmailSender      string
+	SmsAccountID     string
+	SmsKey           string
+	SmsSender        string
+}
+
+// ReadConfig reads the configuration file
+func ReadConfig(config *Configuration) error {
+	// Add matching environment variables - will take precedence over config files.
+	viper.AutomaticEnv()
+
+	// Add default config file search paths in order of decreasing precedence.
+	viper.SetConfigName("amplifier")
+	viper.AddConfigPath("/etc/default/")
```

Comment:
@bquenin Sorry, not sure what happened, but I had updated my comment before saving the review, but I had several comments opened and apparently my final recommendation was lost. Please change to `/etc/atomiq`. Thanks!